videolan/libvlcsharp
# Added minimal Win32 sln

### Description of Change ###

When developing for the first time, users have to install a lot of workloads in their Visual studio setup (Xamarin.Forms, UWP, .Net frameworks targetting packs).

This aims at providing a minimal SLN that contains the projects that can be opened by just having a .NET6 SDK set up.

### Issues Resolved ### 

I can't install the Windows SDK due to my SSD being full.

### API Changes ###
 None, it's not even a target that will be built using the CI

### Platforms Affected ### 

Not applicable from the user's perspective, for windows dev machines.

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###

Open LibVLCSharp.Win32.sln, build it. Open LibVLCSharp.sln and try to build it. Things that worked before should continue to work.

### PR Checklist ###
